exercises/practice/grade-school/GradeSchoolTest.php

```diff
@@ -41,26 +19,96 @@ protected function setUp(): void
         $this->school = new School();
     }
 
-    public function testAddStudent(): void
+    /**
+     * uuid: 9337267f-7793-4b90-9b4a-8e3978408824
+     * @testdox Add a student
+     */
+    public function testAddAStudent(): void
+    {
+        $this->school->add('Aimee', 2);
+        $this->assertCount(1, $this->school->grade(2));
```

Comment:
This is the first breaking change in API we want to make. `add()` returns a `bool`, and we test for it as canonical data wants to have it:

```suggestion
        $this->assertSame(true, $this->school->add('Aimee', 2));
```